.storybook/preview.tsx

```diff
@@ -93,6 +94,12 @@ addParameters({
     options: {},
     manual: false,
   },
+  /**
+   * Defaults to smallest mobile and smallest desktop breakpoints for visual regression testing.
+   * Override on a per-story basis if component stories only need to test one breakpoint,
+   * typically small components that are the same on all views (e.g. Chip, Indicator)
+   */
+  chromatic: { viewports: [BREAKPOINTS.xs, BREAKPOINTS.md] },
```

Comment:
I had originally considered three breakpoints, adding `xl` for a more typical desktop experience instead of `md` which is our bare minimum pixel width breakpoint, but I didn't think it added much value.

Some of our designs don't really respect the xs breakpoint (e.g. the breakpoint is `330px` width but some components have set widths like `327px`, before margin/padding) so the Chromatic screenshots might not be perfect, but it suffices in terms of testing.

Reply:
My preference is to respect the smallest breakpoint we define without creating new stand-alone constants. If something is compatible with `BREAKPOINTS.xs` it would mean support for iPhone 5, which some (albeit a small amount, 1-2%?) of our users have.

I'm aware that Design uses an iPhone X specification but I think it's a reasonable compromise to test at lower dimensions to see that we're not relying on too many hard-coded dimensions. I'm pretty sure the only thing that needs to change for support <375px width is just using `max-width` instead of `width`.